pkg/projects/Microsoft.NETCore/Microsoft.NETCore.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.0.3</Version>
+    <Version>5.0.2</Version>
```

Comment:
Sounds good. Let me know if you have any more concerns on this PR. I'll also wait for Gaurav to comment on the PR.